readme.md

```diff
@@ -218,29 +220,44 @@ Contains libraries that enable interactions with the Device Proviosining service
 Contains libraries that provide modeling and JSON serialization capabilities on top of the raw messaging library.
 
 
-# Long Term Support
+# Releases
 
-The project offers a Long Term Support (LTS) version to allow users that do not need the latest features to be shielded from unwanted changes.
+The C SDK offers releases for new features, security updates, and Long Term Support (LTS). General bug fixes will not receive a separate release. Versioning follows [semantic versioning](https://semver.org/), `x.y.z.` or `major.minor.patch`. Any time the version is updated, it will be tagged `x.y.z`.
 
-A new LTS version will be created every 6 months. The lifetime of an LTS branch is currently planned for one year. LTS branches receive all bug fixes that fall in one of these categories:
+## Features and security updates
 
-- security bugfixes
-- critical bugfixes
+New features and security updates will be released on the main branch. These releases will be tagged using the date formatted `yyyy-mm-dd`. A feature release will bump the `minor` version and reset the `patch` version to 0. A security update will bump the `patch` version only.
 
-No new features will be picked up in an LTS branch.
+## Long Term Support
 
-LTS branches are named lts_*mm*_*yyyy*, where *mm* and *yyyy* are the month and year when the branch was created. An example of such a branch is *lts_07_2017*.
+New LTS releases branch off of main every six months and will be tagged `LTS_<mm_yyyy>_Ref01`. A new LTS release will inherit the existing version from the main branch. LTS branches are named `lts_mm_yyyy` for the month and year the branch was created. The lifetime of an LTS branch is one year, afterwhich the branch is deleted.
 
-## Schedule
+An updated LTS release will occur when a security fix is ported from the main branch. These updated releases will be tagged in the same manor, except for a bumped Ref##, e.g. `LTS_<mm_yyyy>_Ref02`. The `patch` version will also be bumped. No new features and no general bug fixes will be ported to an LTS update.
 
-Below is a table showing the mapping of the LTS branches to the packages released
+### LTS schedule
+
+Below is a table showing the mapping of the LTS branches to the packages released.
 
   | Package | Github Branch | LTS Tag | LTS Status | LTS Start Date | Maintenance End Date | Removal Date |
   | :-----: | :-----------: | :-----: | :--------: | :------------: | :------------------: | :----------: |
-  | vcpkg: 2021-08-11 | lts_07_2021 | LTS_07_2021_Ref01 | Active | 2021-08-11 | 2022-08-11 | 2022-08-11 |
-  | vcpkg: 2021-01-21 | lts_01_2021 | LTS_01_2021_Ref01 | Active | 2021-01-21 | 2022-01-21 | 2022-01-21 | |
+  | vcpkg: 2021-08-18 | lts_07_2021 | LTS_07_2021_Ref01 | Active | 2021-08-11 | 2022-08-11 | 2022-08-11 |
+  | vcpkg: (unreleased) | lts_01_2021 | LTS_01_2021_Ref01 | Active | 2021-01-21 | 2022-01-21 | 2022-01-21 | |
+
+## Release example
```

Comment:
Although detailed and informative, this seems like too extensive. Could it be simplified? Also most of the versioning info should already be known by customers once you inform them we follow semver guidelines.

Reply:
Sure thing. 
Original table (not shown in this PR) had 6 of the 7 columns: package, github branch, lts status, lts start date, maintenance end date, removal date.  (See commit d28aca1d2e195fea3ec1b16cf6d403e378363dab.)

What if we do package, LTS tag (which can be removed once vcpkg is named the same as the LTS tag), lts start date, and maintenance end date? 4, eventually 3 columns. Or is github branch preferred over LTS tag?

Reply:
Sounds great! Thanks.